mbus_packet.cpp

```diff
@@ -261,33 +261,22 @@ uint16_t verifyCrcBytesCmodeB(uint8_t* pByte, uint8_t* pPacket, uint16_t packetS
 {
   uint16_t crc = 0;
   uint16_t i = 0;
-  if (packetSize > 128) {
-    while (i < 126) {
-      crc = crcCalc(crc, pByte[i]);
-      pPacket[i] = pByte[i];
-      ++i;
-    }
-
-    if ((~crc) != (pByte[i] << 8 | pByte[i + 1])) {
+  //Support only for packets with L (lenght byte) < 128 (packetSize < 131)
+  if (packetSize > 131){
       return (PACKET_CRC_ERROR);
-    }
-
-    pPacket[i] = pByte[i];
-    ++i;
-    pPacket[i] = pByte[i];
-    ++i;
-    crc = 0;
   }
-
+  // To compensate incrrect packet length eg. L=33 -> L=31
```

Comment:
Why it is incorrect?

Reply:
Good question. Probably incorrect is not right word here. It's just different to C1 A frame. 
Following below docu, L-field in C1 B frame includes 2 bytes of CRC checksum. Which is not included in C1 A frame. 
![image](https://github.com/SzczepanLeon/wMbus-lib/assets/149330119/372d1cd9-3b7e-41bc-9d0c-b56933a51a9c)
https://www.st.com/resource/en/application_note/an4772-wmbus-2013-firmware-stack-overview-stmicroelectronics.pdf